app/assets/javascripts/admin/util.js.erb

```diff
@@ -1,6 +1,6 @@
 $(document).ready(function() {
   $('.datetimepicker').datetimepicker({
-    dateFormat: Spree.translations.date_picker,
+    dateFormat: "yy-mm-dd",
```

Comment:
Mh, OK then, maybe you could add this findings as documentation in the code. will be helpful to understand the change in the future